libtransmission/port-forwarding.cc

```diff
@@ -117,7 +124,7 @@ static void set_evtimer_from_status(tr_shared* s)
         /* if we're mapped, everything is fine... check back in 20 minutes
          * to renew the port forwarding if it's expired */
         s->doPortCheck = true;
-        sec = 60 * 20;
+        sec = (int)(s->natpmp->renew_time - time(NULL));
```

Comment:
- code no longer matches the comment above; comment should be updated to match e.g. "check back at `renew_time to renew the port forwarding if it's expired`

- use `tr_time()` instead of `time(NULL)`